unit-tests/run-unit-tests.py

```diff
@@ -111,7 +113,7 @@ def is_executable(path_to_test):
 
 # wrapper function for subprocess.run
 def subprocess_run(cmd, stdout = None):
-    log.d( 'running:', cmd )
+    log.d( '    running:', cmd )

```

Comment:
Why did you add spaces?

Reply:
So visually it would look better:

```
-D- found test-new-sys-path
-D-     script: C:\work\git\lrs\unit-tests\new/test-sys-path.py
-D-     running: ['C:\\Program Files (x86)\\Microsoft Visual Studio\\Shared\\Python37_64\\python.exe', '-S', 'C:\\work\\git\\lrs\\unit-tests\\new/test-sys-path.py', '--debug', '--color']
...
```